.github/workflows/go-apply-patch.sh

```diff
@@ -9,6 +9,11 @@ prepare_git_repo
 # create a mapping between short version and new version, e.g. 1.16 -> 1.16.3
 declare -A VERSIONS
 for version_new in ${VERSIONS_NEW}; do
+  if ! check_remote_branch "go-${version_new}-${TARGET_BRANCH}"; then
+      echo 'UPDATE_NEEDED=0' >>"${GITHUB_OUTPUT}"
+      exit 0
+  fi
```

Comment:
This won't work. You want to add this check after `branch_name` is set below this loop, so you could do `if check_remote_branch "${branch_name}"; …`.

Reply:
Fixed, thanks.